### PR TITLE
Add details about response object

### PR DESCRIPTION
**Pull request by veustepa**
Currently, /joinedTeams API call doesn't return values for classSettings properties through it contains the properties in the response object, this commit will keep the user addressed about this context and adds information to see where the values can be fetched from

### CODE REVIEW

File: api-reference/beta/api/user-list-joinedteams.md [hunk 1]
```diff
@@ -53,8 +53,10 @@ Do not supply a request body for this method.
 
 If successful, this method returns a `200 OK` response code and collection of [team](../resources/team.md) objects in the response body.
 
+> **Note:** Currently, this API call returns values only for the properties **id**, **displayName**, and **description** of a [team](../resources/team.md). To get information on all properties, use [Get team](/graph/team-get)].
+
 ## Example
-### Request
+##### Request
```
Comment (AkJo) [inline api-reference/beta/api/user-list-joinedteams.md:59]: Why are we reducing the size here?

Comment (veustepa) [reply api-reference/beta/api/user-list-joinedteams.md:59]: I accidentally changed it to be consistent with v1.0 document, will fix that.

File: api-reference/beta/api/user-list-joinedteams.md [hunk 2]
```diff
@@ -98,14 +100,30 @@ Content-type: application/json
...
   "value": [
     {
-      "id": "31aa74dd-dd65-43ac-8c4e-0ec1ae5a8ee1"
+      "id": "172b0cce-e65d-44ce-9a49-91d9f2e8493a",
+      "createdDateTime": null,
+      "displayName": "Contoso Team",
+      "description": "This is a Contoso team, used to showcase the range of properties supported by this API",
+      "internalId": null,
+      "classification": null,
+      "specialization": null,
```
Comment (AkJo) [inline api-reference/beta/api/user-list-joinedteams.md:109]: Please include only the properties that are valid - which are id, displayName and description.

Comment (AkJo) [reply api-reference/beta/api/user-list-joinedteams.md:109]: This applies both to v1 and beta.

File: api-reference/beta/api/user-list-joinedteams.md [hunk 3]
```diff
@@ -53,6 +53,8 @@ Do not supply a request body for this method.
 
 If successful, this method returns a `200 OK` response code and collection of [team](../resources/team.md) objects in the response body.
 
+> **Note:** Currently, this API call returns values only for the properties **id**, **displayName**, and **description** of a [team](../resources/team.md). To get information on all properties, use [Get team](/graph/team-get)].
```
Comment (AkJo) [inline api-reference/beta/api/user-list-joinedteams.md:56]: Extra bracket

File: api-reference/beta/api/user-list-joinedteams.md (hunk 2 above)
Comment (veustepa) [reply api-reference/beta/api/user-list-joinedteams.md]: Fixed

File: api-reference/beta/api/user-list-joinedteams.md (hunk 3 above)
Comment (veustepa) [reply api-reference/beta/api/user-list-joinedteams.md]: Removed

File: concepts/known-issues.md [hunk 4]
```diff
@@ -351,6 +351,9 @@ The filter query to get members of a team based on their roles `GET /teams/team-
 ### Missing properties for chat members
 In certain instances, the `tenantId` / `email` / `displayName` property for the individual members of a chat might not be populated on a `GET /chats/chat-id/members` or `GET /chats/chat-id/members/membership-id` request.
 
+### Unable to return all values for properties for a user joined teams
```
Comment (AkJo) [inline concepts/known-issues.md:354]: Missing properties in the list of teams that a user has joined

File: api-reference/beta/api/user-list-joinedteams.md [hunk 5]
```diff
@@ -53,6 +53,9 @@ Do not supply a request body for this method.
 
 If successful, this method returns a `200 OK` response code and collection of [team](../resources/team.md) objects in the response body.
 
+> [!Note]
+> Currently, this API call returns values only for the properties **id**, **displayName**, and **description** of a [team](../resources/team.md). To get information on all properties, use [Get team](../api/team-get.md). For details, see [known issues](/graph/known-issues#unable-to-return-all-values-for-properties-for-a-user-joined-teams).
```
Comment (AkJo) [inline api-reference/beta/api/user-list-joinedteams.md:57]: Remember to update this when you make the corresponding change in known issues file.

File: api-reference/v1.0/api/user-list-joinedteams.md [hunk 6]
```diff
@@ -50,7 +50,11 @@ Do not supply a request body for this method.
 
 ## Response
 
-If successful, this method returns a `200 OK` response code and collection of [group](../resources/group.md) objects in the response body.
+If successful, this method returns a `200 OK` response code and collection of [team](../resources/team.md) objects in the response body.
+
+> [!Note]
+> Currently, this API call returns values only for the properties **id**, **displayName**, and **description** of a [team](../resources/team.md). To get information on all properties, use [Get team](../api/team-get.md). For details, see [known issues](/graph/known-issues#unable-to-return-all-values-for-properties-for-a-user-joined-teams).
```
Comment (AkJo) [inline api-reference/v1.0/api/user-list-joinedteams.md:56]: Remember to update this when you make the corresponding change in known issues file.

File: api-reference/beta/api/user-list-joinedteams.md (hunk 5 above)
Comment (AkJo) [inline api-reference/beta/api/user-list-joinedteams.md:57]: Will need some word-smithing by @Lauragra / @JarbasHorst

**Review by AkJo — APPROVED** [review]
Please apply the mentioned feedback.

File: api-reference/beta/api/user-list-joinedteams.md (hunk 5 above)
Comment (Lauragra) [reply api-reference/beta/api/user-list-joinedteams.md]: Made some edits to the updated text.

**Pull request merged**